commons/src/main/kotlin/com/simplemobiletools/commons/compose/screens/ContributorsScreen.kt

```diff
@@ -0,0 +1,148 @@
+            }
+            item {
+                SettingsGroupTitle {
+                    SettingsTitleTextComponent(text = stringResource(id = R.string.translation), modifier = startingPadding)
+                }
+            }
+            items(contributors, key = { it.contributorsId.plus(it.iconId).plus(it.labelId) }) {
+                ContributorItem(
+                    languageContributor = it
+                )
+            }
+            if (showContributorsLabel) {
+                item {
+                    SettingsListItem(
+                        icon = R.drawable.ic_heart_vector,
+                        text = {
+                            val source = stringResource(id = R.string.contributors_label)
+                            LinkifyText {
+                                stringFromHTML(source)
+                            }
+                        },
+                        tint = MaterialTheme.colorScheme.onSurface
+                    )
+                }
+                item {
+                    Spacer(modifier = Modifier.padding(bottom = 8.dp))
+                }
+            }
+        })
+}
+
+@Composable
+private fun ContributorItem(
+    modifier: Modifier = Modifier,
+    languageContributor: LanguageContributor
+) {
+    ListItem(
+        headlineContent = {
+            Text(
+                text = stringResource(id = languageContributor.labelId),
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .then(modifier)
+            )
+        },
+        leadingContent = {
+            val imageSize = Modifier
+                .size(48.dp)
+                .padding(8.dp)
+            Image(
+                modifier = imageSize,
+                painter = painterResource(id = languageContributor.iconId),
+                contentDescription = stringResource(id = languageContributor.contributorsId),
+            )
+        },
+        modifier = Modifier
+            .fillMaxWidth(),
+        supportingContent = {
+            Text(
+                text = stringResource(id = languageContributor.contributorsId),
+                modifier = Modifier
+                    .fillMaxWidth()
+            )
+        }
+    )
+}
+
+@Composable
+@MyDevices
+private fun ContributorsScreenPreview() {
+    AppThemeSurface {
+        ContributorsScreen(
+            goBack = {},
+            contributors = listOf(
+                LanguageContributor(R.drawable.ic_flag_arabic_vector, R.string.translation_arabic, R.string.translators_arabic),
+                LanguageContributor(R.drawable.ic_flag_azerbaijani_vector, R.string.translation_azerbaijani, R.string.translators_azerbaijani),
+                LanguageContributor(R.drawable.ic_flag_bengali_vector, R.string.translation_bengali, R.string.translators_bengali),
+                LanguageContributor(R.drawable.ic_flag_catalan_vector, R.string.translation_catalan, R.string.translators_catalan),
```

Comment:
wheres the rest?

Reply:
I just copy pasted the same list that was there, am I missing something?

Reply:
Although there's this logic

` val licenseMask = remember { intent.getLongExtra(APP_LICENSES, 0) or LICENSE_KOTLIN }
            val thirdPartyLicenses = remember { initLicenses().filter { licenseMask and it.id != 0L }.toImmutableList() }`

which only displays 2 of them if not launched with an intent extra